backend/cellguide/pipeline/ontology_tree/tree_builder.py

```diff
@@ -719,7 +722,7 @@ def _build_parents_per_node(self, node, all_parents=None) -> Dict[str, list[str]
             self._build_parents_per_node(child, all_parents)
         return all_parents
 
-    def _get_deepcopy_of_ontology_graph(self) -> Dict[str, Any]:
+    def _get_deepcopy_of_ontology_graph(self) -> OntologyTree:
```

Comment:
for a python dataclass i believe you would want to update this to be
```
return copy.deepcopy(self.ontology_graph)
```
https://docs.python.org/3/library/copy.html